src/compas/utilities/functions.py

```diff
@@ -19,6 +22,58 @@ def fibonacci(n, memo={}):
         memo[n] = fibonacci(n - 2, memo) + fibonacci(n - 1, memo)
     return memo[n]
 
+def avrg(list):
+    """Average of a list.
+
+    Parameters
+    ----------
+    list : list
+        List of values.
+
+    Returns
+    -------
+    float
+        The mean value.
+
+    """
+
+    return sum(list) / float(len(list))
+
+def var(list):
+    """Variance of a list.
+
+    Parameters
+    ----------
+    list : list
+        List of values.
+
+    Returns
+    -------
+    float
+        The variance value.
+
+    """
+
+    m = avrg(list)
+
+    return (sum([(i - m) ** 2 for i in list]) / float(len(list))) ** .5
+
+def st_dev(list):
```

Comment:
Same as above, I'd stick to numpy naming and call this `std`.

Reply:
i would just call this standard_deviation